### DISCUSSION TITLE
Getting started with mods - link broken

### DISCUSSION THREAD

**Discussion by Kellei2983** [mbround18/valheim-docker#411] [General]
Hi, I wanted to add mods to my server, but the link with info about how to run them is broken.

**Comment by bigsby-exe** [comment]
Take a look here: [Getting started with mods](https://github.com/mbround18/valheim-docker/blob/main/docs/tutorials/getting_started_with_mods.md)

**Reply by Kellei2983** [reply]
Thanks, this one works, I have create a pull request to update the link in readme.
